projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -259,13 +299,40 @@ public function checkout_process() {
 				continue;
 			}
 
-			$this->record_event(
+			$properties = $this->record_event(
 				'woocommerceanalytics_product_checkout',
 				$product->get_id(),
 				array(
-					'pq' => $cart_item['quantity'],
-				)
+					'pq'               => $cart_item['quantity'],
+					'payment_options'  => $enabled_payment_options,
+					'device'           => wp_is_mobile() ? 'mobile' : 'desktop',
+					'guest_checkout'   => 'Yes' === $guest_checkout ? 'Yes' : 'No',
+					'create_account'   => 'Yes' === $create_account ? 'Yes' : 'No',
```

Comment:
I think we should make them as clear as possible to start – will propose tweaks in my review :)

Also I think it's worth getting a review from data team (more important than my review really). Ideally it would be good to have a P2 outlining exactly what questions we need to answer, so we can ensure we are tracking the right info, and nothing redundant.